aws/table_aws_wellarchitected_check_summary.go

```diff
@@ -0,0 +1,219 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedCheckSummary(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_check_summary",
+		Description: "AWS Well-Architected Summary",
```

Comment:
Should it not be `AWS Well-Architected Check Summary` instead of `AWS Well-Architected Summary`?